importer/tests/test_views.py

```diff
@@ -20,17 +23,113 @@
 class TestViews(LoreTestCase):
     """Hit each view."""
 
+    import_url = "/repositories/{0}/import/"
+    repository_url = "/repositories/{0}/"
```

Comment:
the strings are used with at least 2 different slugs, so I guess I still need to format it